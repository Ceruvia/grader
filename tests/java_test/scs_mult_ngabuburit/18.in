4 2 7 2 2 5 3 4 1 4 1 0 1 4 3 3 2 