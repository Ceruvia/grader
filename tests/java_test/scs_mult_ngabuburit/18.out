Pintu dibuka!
Pintu dibuka!
Sudah jenuh! Lakukan aktivitas lain
Sudah jenuh! Lakukan aktivitas lain
Pergi ke luar rumah!
Berlari di gasibu sejauh 10 km!
Membeli kolak!
Sudah jenuh! Lakukan aktivitas lain
Laptop dinyalakan!
Menggunakan laptop untuk ngoding gacor
Menggunakan laptop untuk ngoding gacor
Sudah jenuh! Lakukan aktivitas lain
Laptop dinyalakan!
Menggunakan laptop untuk ngoding gacor
Menggunakan laptop untuk ngoding gacor
Sudah jenuh! Lakukan aktivitas lain
Pergi ke luar rumah!
Berlari di gasibu sejauh 10 km!
Membeli kolak!
